core-java-8/src/main/java/com/baeldung/shufflingcollections/ShufflingCollections.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.shufflingcollections;
+
+import java.util.*;
+
+public class ShufflingCollections {
```

Comment:
Do we still need this class? All the code is already in the test class.

Reply:
No, we don't need this class. I wasn't sure if sample code in tests alone is sufficient.

Reply:
Removed ShufflingCollections class.